src/java.base/share/classes/java/lang/Enum.java

```diff
@@ -59,7 +59,7 @@
  * {@linkplain java.util.EnumSet set} and {@linkplain
  * java.util.EnumMap map} implementations are available.
  *
- * @param <E> The enum type subclass
+ * @param <E> The enum class subclass
```

Comment:
I wonder about this one, given that `<E>` is a type. It sounds strange to me - even though I understand that what is meant is that this type should point to a class (a subclass of `java.lang.Enum`).

Reply:
Makes sense. I suggest the following instead: `The type of enum subclass`.

Reply:
Maybe: "`The type of the enum subclass.`" would sound better - or just "`The enum subclass`".